hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/OMConfigKeys.java

```diff
@@ -54,6 +54,8 @@ private OMConfigKeys() {
   public static final String OZONE_OM_BIND_HOST_DEFAULT =
       "0.0.0.0";
   public static final int OZONE_OM_PORT_DEFAULT = 9862;
+  public static final String OZONE_OM_GRPC_PORT_KEY =
+      "ozone.s3g.grpc.port";
```

Comment:
Minor: ozone.s3g.grpc.port -> ozone.om.grpc.port

Can we name this generic like ozone.om.grpc.port instead of s3g, in future if any clients use it, naming will not cause confusion

Reply:
After reading code i think this should be renamed as ozone.om.grpc.port, as GrpcOzoneManagerServer defined config key in that way too.

Reply:
@bharatviswa504 - latest commit contains changes to ozone.s3g.grpc.port -> ozone.om.grpc.port in OMConfigKeys.  This impacts the TestOzoneConfigurationFields integration test as key is not in site xml file, so added line to ignore key.